index.js

```diff
@@ -0,0 +1,66 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Styling is on point and followed guideline provided by the rubric well. Here are few feedbacks on functionalities: 
1. Comment at the bottom need to show movie's name which user is commenting on. 
2. Select and options need significant improvement and change of codes to make them functional. I think you got the right idea as to how to do it. Please re-visit this assessment to fix those problems. 
3. Minor feedback would be to create smaller functions with specialized tasks to make your code clean and DRY. 
4. Please explore more into use of "event" and "event listener" which will help you solve the issue of displaying a single movie info based on user's choice. 

As previously mentioned, please re-visit this assessment to make functionalities complete. Things that we do in this assessment will be more relevant as we go into unit 3 and 4, and it's important to have good understanding of DOM manipulation and creating working UI. Let me know if you need any help or wants to schedule a review session on materials.